src/main/java/io/stargate/sgv2/jsonapi/service/shredding/collections/DocumentId.java

```diff
@@ -82,8 +82,8 @@ static DocumentId fromJson(JsonNode node) {
             "unrecognized JSON extension type '%s'", node.fieldNames().next());
     }
     throw ErrorCodeV1.SHRED_BAD_DOCID_TYPE.toApiException(
-        "Document Id must be a JSON String, Number, Boolean, EJSON-Encoded Date Object or NULL instead got %s",
-        node.getNodeType());
+        "Document Id must be a JSON String, Number, Boolean, EJSON-Encoded Date Object or NULL instead got %s: %s",
```

Comment:
Actual change to include JSON value so logs show what actually was the mismatch (not just "there was a JSON Object")